bin/commander.js

```diff
@@ -237,8 +237,18 @@ async function updateThePluginStableVersion( version, abortMessage ) {
  */
 async function runCleanLocalCloneStep( abortMessage ) {
 	await runStep( 'Cleaning the temporary folder', abortMessage, async () => {
-		await rimraf( gitWorkingDirectoryPath );
-		await rimraf( svnWorkingDirectoryPath );
+		await Promise.all( [
+			gitWorkingDirectoryPath,
+			svnWorkingDirectoryPath,
+		].map( async ( directoryPath ) => {
+			if ( fs.existsSync( directoryPath ) ) {
+				await rimraf( directoryPath, ( err ) => {
```

Comment:
They do the same trick in `rimraf` in their bin implementation: 
https://github.com/isaacs/rimraf/blob/d82bc81f251ba2cc86dc26361a820631091b3e9e/bin.js#L44-L48

Reply:
It fixes the following error I observed locally:

```
The following error happened during the "Cleaning the temporary folder" step:
 AssertionError [ERR_ASSERTION]: rimraf: callback function required
    at rimraf (/Users/gziolo/PhpstormProjects/gutenberg/node_modules/rimraf/rimraf.js:52:10)
```